code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -30,7 +30,7 @@ obj/item/weapon/gun/energy/retro
 	w_class = 3
 	projectile_type = /obj/item/projectile/beam
 	fire_delay = 10 //old technology
-
+	pin = null

```

Comment:
aaaaaaaaaaaaaaaaaaaaaaaaa

Seriously you gotta stop doing this shit. Add a new line after this.

Reply:
lol I don't even need that